packages/vite/src/node/plugins/css.ts

```diff
@@ -158,6 +159,12 @@ export function cssPlugin(config: ResolvedConfig): Plugin {
         }
         const resolved = await resolveUrl(url, importer)
         if (resolved) {
+          if (config?.css?.publicPath) {
+            return [
+              config.css.publicPath,
+              path.posix.relative(config.root, resolved)
+            ].join('')
```

Comment:
Haha ^^' sorry, I think you misunderstand me a bit. I just asked if the braces are necessary or not. If they are and they are there so the wrapping is correct, then it is okay.
You new solution is a bit worse then before, cause it adds runtime overhead (I now these are nanoseconds XD, but a join costs more then a simple concat)

I just expected something like 

```js
return config.css.publicPath + path.posix.relative(config.root, resolved)
```

but okay, just revert it 🙂 
Sorry for miscommunication

Reply:
haha, ok, I revert it~.